app/domain/conjur/fetch_trusted_proxies.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+require 'command_class'
+
+module Conjur
+
+  POLICY_ID = "settings"
+  HOST_NAME = "trusted_proxies"
+
+  FetchTrustedProxies ||= CommandClass.new(
+      dependencies: {
+          role_cls: ::Role,
+          validate_account_exists: ::Authentication::Security::ValidateAccountExists.new,
+          logger: Rails.logger
+      },
+      inputs: %i(account)
+  ) do
+
+    def call
+      validate_account_exists
+      fetch_trusted_proxies_list
+      delete_duplications_in_list
+      trusted_proxies_list
+    rescue => e
+      raise Errors::Conjur::TrustedProxiesFetchFailed.new(
+          host_id,
+          e.inspect
+      )
+    end
+
+    private
+
+    def validate_account_exists
+      @validate_account_exists.(
+          account: @account
+      )
+    end
+
+    def fetch_trusted_proxies_list
+      @logger.debug(LogMessages::Conjur::FetchingTrustedProxies.new(host_id))
+      @trusted_proxies_list = role.restricted_to
+      @trusted_proxies_list ||= []
```

Comment:
can you add a comment to explain this line? Do you initialize to an empty array in case the trusted_proxies role doesn't have a `restricted_to` field?